docs/architecture/modernize-desktop/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Modernizing Desktop Apps on Windows 10 with .NET Core 3
+description: Learn how to modernize existing desktop apps with .NET Core 3.0
+author: nishanil
+ms.author: nanil
+ms.date: 04/14/2020
+---
+# Modernizing Desktop Apps on Windows 10 with .NET Core 3
+
+![Screenshot that shows the modernize desktop apps e-book cover.](./media/modernizing-existing-desktop-apps-ebook-cover.png)
+
+PUBLISHED BY
+
+Microsoft Developer Division, .NET, and Visual Studio product teams
+
+A division of Microsoft Corporation
+
+One Microsoft Way
+
+Redmond, Washington 98052-6399
+
+Copyright © 2020 by Microsoft Corporation
+
+All rights reserved. No part of the contents of this book may be reproduced or transmitted in any form or by any means without the written permission of the publisher.
+
+This book is provided "as-is" and expresses the author's views and opinions. The views, opinions and information expressed in this book, including URL and other Internet website references, may change without notice.
+
+Some examples depicted herein are provided for illustration only and are fictitious. No real association or connection is intended or should be inferred.
+
+Microsoft and the trademarks listed at <https://www.microsoft.com> on the "Trademarks" webpage are trademarks of the Microsoft group of companies.
+
+Mac and macOS are trademarks of Apple Inc.
+
+All other marks and logos are property of their respective owners.
+
+Authors:
+
+TODO: Add authors here.
+
+## Introduction
+
+TODO: Add intro
+
+## Who should use the book
+
+TODO: Add details
```

Comment:
I think some of this todo's are in chapter 1. Should we move it here?

Reply:
Yes, I will ask @OliaG  to move them here.